consul-secure-acl-vault/step3.md

```diff
@@ -10,13 +10,13 @@ Once the ACL system is initialized, if you check Consul logs again you should fi
  [WARN]  agent: Coordinate update blocked by ACLs: accessorID=00000000-0000-0000-0000-000000000002
 ```
 
-These warnings are ACL logs from which you can infer the following: 
-- one node is trying to update its info in your Consul datacenter and gets denied access by the ACL system.
-- the node is not using any token to perform the request (because `00000000-0000-0000-0000-000000000002` is the anonymous policy used when no token is presented)
+These warnings indicate that: 
+- the agent is trying to update its info in your Consul datacenter, but denied by the ACL system because it does not have the correct permissions.
```

Comment:
```suggestion
- the agent is trying to update its info in your Consul datacenter, but has been denied by the ACL system because it does not have the correct permissions.
```